spec/helpers/tasklist_helper_spec.rb

```diff
@@ -165,7 +165,7 @@
 
           it "returns relevant text" do
             expect(review_log_text(lettings_log)).to eq(
-              "You can #{govuk_link_to 'review and make changes to this log', review_lettings_log_path(lettings_log)} until 9 June 2024.".html_safe,
+              "You can #{govuk_link_to 'review and make changes to this log', review_lettings_log_path(lettings_log)} until 7 June 2024.".html_safe,
```

Comment:
Though I'm guessing there isn't something in place for this, I don't like the idea of dates being plain text like this, perhaps some rudimentary CMS or JSON for storing significant dates as well as a description of what each date represents would be good for this sort of thing.